src/odemis/driver/technolution.py

```diff
@@ -1070,10 +1068,12 @@ def _assembleMegafieldMetadata(self):
         X_descan_setpoints = self._descanner.getXAcqSetpoints()
         Y_descan_setpoints = self._descanner.getYAcqSetpoints()
 
+        storage_directory = os.path.join(urlparse(self.parent.externalStorageURL.value).path, self.subdirectory.value)
```

Comment:
Drop this. Just `storage_directory = self.subdirectory.value`. Then, it'll be possible to indicate the whole base URL in externalStorageURL.